packages/expo-font/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### 🎉 New features
 
+- add `getLoadedFonts()` function ([#30431](https://github.com/expo/expo/pull/30431) by [@vonovak](https://github.com/vonovak))
+- Allow woff and woff2 format on iOS with the config plugin. ([#30220](https://github.com/expo/expo/pull/30220) by [@titozzz](https://github/Titozzz))
```

Comment:
Think this was from a merge conflict. This entry is present in `12.0.8 - 2024-07-11` below.
```suggestion
```